docs/SemanticARC.md

```diff
@@ -194,7 +244,11 @@ In this case, since the apply's second argument must be @owned, a simple use-def
       return %result : $()
     }
 
-While this may look correct to the naked eye, it is actually incorrect even in SIL today. This is because in SIL today, switch_enum always takes arguments at +1! Yet, in the IR there is no indication of the problem and the code will compile! Now let us update the IR given semantic ARC:
+While this may look correct to the naked eye, it is actually incorrect even in SIL today. This is because in SIL today, switch_enum always takes arguments at +1!
+
+[EE] Hm... no. switch_enum doesn't take it's argument as owned.
```

Comment:
When compiling this:

final class X {
  var v : Int = 0
}
func testit(_ x: X?) {
  if let xx = x {
    xx.v = 27
  }
}

there is a release_value %0 in the function. So the switch_enum %0 does not take it.